packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -182,7 +182,7 @@ const ButtonCore: React.ForwardRefExoticComponent<
                                 sharedStyles.endIconWrapperTertiary,
                             (focused || hovered) &&
                                 kind !== "primary" &&
-                                sharedStyles.iconWrapperSecondaryHovered,
+                                buttonStyles.iconWrapperHovered,
```

Comment:
I've moved this to the function where we generate style colors dynamically to be able to support `default` (`progressive`) and `destructive` colors.